### DISCUSSION TITLE
Get 'Display Output is not active.' on IpadOS 18 for Ubuntu 18 ARM64 server

### DISCUSSION THREAD

**Discussion by artificialhumanbeing** [utmapp/UTM#6884] [UTM Help (iOS)]
After select “install Ubuntu server” on GNU GRUB, get 'Display Output is not active.'

**Comment by artificialhumanbeing** [comment]
UTM Version: utm se 4.6.2
IpadOS Version: 18.1
Mac Chip : M2

**Reply by Pradnyana28** [reply]
Got the same issue too

**Reply by jobam** [reply]
Same issue too

**Comment by devin-s07** [comment]
Same exact issue here, did anyone find a solution, or is this just gonna go unsolved like the other 876 issues in this repo?

**Comment by redlfox** [comment]
I had the same problem booting my Debian 10 arm64 from offical cloud image on iPad mini 6 iPadOS 17.6 on UTM SE 4.6.2. I always failed to boot and got either 'display output is not active' or a black screen. But i managed to boot it successfully with these settings: Enable UEFI Boot in QEMU settings and switch "Emulated Display Card" to "virtio-gpu-gl-pci" in Display settings. Not sure if this will work on other linux arm64 OS. The qcow2 i used for Debian 10 arm64: https://cloud.debian.org/images/cloud/buster/daily/latest/debian-10-nocloud-arm64-daily.qcow2
